modules/microshift-opt-out-telemetry.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// microshift_support/microshift-remote-cluster-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-opt-out-telemetry_{context}"]
+= Opting out of Telemetry for {microshift-short}
+
+If your cluster is not connected to a network, or you simply do not want telemetry gathered, you can easily opt out of telemetry by disabling the parameter in the {microshift-short} configuration file.
+
+.Prerequisties
+
+* You installed {oc-first}.
+* You have root access to the cluster.
+
+.Procedure
+
+. If you have not done so, make a copy of the provided `config.yaml.default` file in the `/etc/microshift/` directory, renaming it `config.yaml`.
+
+. Keep the new {microshift-short} `config.yaml` in the `/etc/microshift/` directory. Your `config.yaml` file is read every time the {microshift-short} service starts.
++
+[NOTE]
+====
+After you create it, the `config.yaml` file takes precedence over built-in settings.
+====
+
+. Optional: Use a configuration snippet if you are using an existing {microshift-short} YAML. See "Using configuration snippets" in the Additional resources section for more information.
```

Comment:
"For more information, see "Using configuration snippets" in the Additional resources section.